generator/pubspec.yaml

```diff
@@ -1,24 +1,24 @@
 name: retrofit_generator
 description: retrofit generator is an dio client generator using source_gen and inspired by Chopper and Retrofit.
-version: 4.0.1
+version: 4.1.0
 
 homepage: https://mings.in/retrofit.dart/
 repository: https://github.com/trevorwang/retrofit.dart/
 
 environment:
-  sdk: '>=2.14.0 <3.0.0'
+  sdk: '>=2.17.0 <3.0.0'
 
 dependencies:
-  dio: ^4.0.1
-  source_gen: ^1.0.1
-  built_collection: ^5.0.0
-  code_builder: ^4.0.0
-  tuple: ^2.0.0
+  analyzer: '>=4.6.0 <6.0.0'
+  build: ^2.3.1
+  built_collection: ^5.1.1
+  code_builder: '>=4.0.0 <4.3.0'
```

Comment:
This line

Reply:
OK   

code_builder can be updated in future release.